### DISCUSSION TITLE
Listing low stock but excluding items already on order?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#4508] [Q&A] [answered]
When I'm running low on stock I create a purchase order. Items are then marked with a trolley with the on order status. All good there.  (When I item is received I mark the PO as received and the stock count updates)

When I view items with low stock is there a way to exclude the items already on order?  I keep seeing all the low stock items and think I need to re-order, only then to see the order status.

**Comment by matmair** [comment]
Not that I am aware of it right now.

**Comment by StormTrooper** [comment, marked as answer]
I found a workaround, I created a PHP script to query the parts api and looped though the values with ordering=0 (ie nothing on order)

**Reply by matmair** [reply]
Not sure that is a permanent solution but good for now.
